[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/adapters-and-accelerators/install-biztalk-adapter-pack-2013-r2-and-2013.md

> | **BAP 2013 R2**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)] R2<br/>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>Windows 8.1<br/>Windows 7 SP1</li><li>[!INCLUDE[dotnet451](../includes/dotnet451-md.md)]</li><li> Visual Studio 2013</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li></ul> **BAP 2013**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>[!INCLUDE[btsWinSvr2k8R2](../includes/btswinsvr2k8r2-md.md)] SP1<br/>Windows 8<br/>Windows 7 SP1</li><li>Microsoft [!INCLUDE[dotnet45](../includes/dotnet45-md.md)]</li><li>Visual Studio 2012</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li></ul> | **BAP 2013 R2**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)] R2<br/>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>Windows 8.1<br/>Windows 7 SP1</li><li>[!INCLUDE[dotnet451](../includes/dotnet451-md.md)]</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li><li>Internet Information Services (IIS) version that comes with the operating system. KB 224609 lists the versions.</li></ul>**BAP 2013**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>[!INCLUDE[btsWinSvr2k8R2](../includes/btswinsvr2k8r2-md.md)] SP1<br/>Windows 8<br/>Windows 7 SP1</li><li>Microsoft [!INCLUDE[dotnet45](../includes/dotnet45-md.md)]</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li><li>Internet Information Services (IIS) version that comes with the operating system. KB 224609 lists the versions.</li></ul> |            Microsoft Office SharePoint Server Software Development Kit (SDK)             | Microsoft Office Servers Infrastructure Update. Download [here](https://go.microsoft.com/fwlink/?LinkId=128344). |

Reviewer: Never use "here" as link text:

```suggestion
| **BAP 2013 R2**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)] R2<br/>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>Windows 8.1<br/>Windows 7 SP1</li><li>[!INCLUDE[dotnet451](../includes/dotnet451-md.md)]</li><li> Visual Studio 2013</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li></ul> **BAP 2013**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>[!INCLUDE[btsWinSvr2k8R2](../includes/btswinsvr2k8r2-md.md)] SP1<br/>Windows 8<br/>Windows 7 SP1</li><li>Microsoft [!INCLUDE[dotnet45](../includes/dotnet45-md.md)]</li><li>Visual Studio 2012</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li></ul> | **BAP 2013 R2**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)] R2<br/>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>Windows 8.1<br/>Windows 7 SP1</li><li>[!INCLUDE[dotnet451](../includes/dotnet451-md.md)]</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li><li>Internet Information Services (IIS) version that comes with the operating system. KB 224609 lists the versions.</li></ul>**BAP 2013**:<ul><li>[!INCLUDE[btsWinSrv2k12](../includes/btswinsrv2k12-md.md)]<br/>[!INCLUDE[btsWinSvr2k8R2](../includes/btswinsvr2k8r2-md.md)] SP1<br/>Windows 8<br/>Windows 7 SP1</li><li>Microsoft [!INCLUDE[dotnet45](../includes/dotnet45-md.md)]</li><li>[!INCLUDE[afproductnameshort](../includes/afproductnameshort-md.md)]</li><li>Microsoft [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)]</li><li>Respective enterprise application clients and associated software. See [Supported Enterprise Application Versions](#BKMK_SuppLOB).</li><li>Internet Information Services (IIS) version that comes with the operating system. KB 224609 lists the versions.</li></ul> | Microsoft Office SharePoint Server Software Development Kit (SDK) | Microsoft Office Servers Infrastructure Update, To download, visit the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=128344). |
```

   This ZIP file contains Microsoft run-time DLLs, which you can download from [SAP Support - Software Downloads](https://support.sap.com/en/my-support/software-downloads.html). For more information, see SNOTE<sup>1</sup> 684106. This link has an “Attachments” option from where you can download the package.

- Microsoft Visual C++ run-time DLLs required for SAP 7.1 client are available from the following links:
  - For 32-bit SAP 7.1 client:  Download **Vcredist_x86.exe** from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=107086).
  - For 64-bit SAP 7.1 client: Download **Vcredist_x64.exe** from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=107087).

- If you use SAP Secure Network Communications (SNC) to connect to an SAP system, you must also have the relevant DLLs from SAP. These DLLs are different for 32-bit and 64-bit platforms and are available with SNOTE<sup>1</sup> 352295. You can download the DLLs from [here](https://go.microsoft.com/fwlink/?LinkId=104032). This link has an “Attachments” option from where you can download the package. The names of the DLLs are:

>   - **For 32-bit**: *gsskrb5.dll, gssntlm.dll*

Reviewer: ```suggestion
  - For 32-bit: **gsskrb5.dll**, **gssntlm.dll**
```

>   - **For 64-bit x86**: *gx64krb5.dll, gx64ntlm.dll*

Reviewer: ```suggestion
  - For 64-bit x86: **gx64krb5.dll**, **gx64ntlm.dll**
```

> Applies to BizTalk Adapter Pack 2013 only.

Reviewer: ```suggestion
The following information applies only to BizTalk Adapter Pack 2013.
```

> - **SAP RFC SDK 6.40 UNICODE**. This is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available [here](https://go.microsoft.com/fwlink/?LinkId=94691). After you have downloaded and extracted the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders to the relevant location mentioned preceding this table.

Reviewer: The FW link was broken, fixed with replacement:

```suggestion
- **SAP RFC SDK 6.40 UNICODE**. This is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available at the [SAP Support Portal](https://support.sap.com/en/index.html). After you have downloaded and extracted the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders to the relevant location mentioned preceding this table.
```

> - DLLs are available from SAP as part of **R3DLLINST.zip**. This contains Microsoft run-time DLLs and can be downloaded from the SAP site. See SNOTE<sup>1</sup> 684106 for more information. You can download the .zip file from [here](https://go.microsoft.com/fwlink/?LinkId=94693). This link has an “Attachments” option from where you can download the package.

Reviewer: The FW link just goes to the SAP support sign-in page.
```suggestion
- DLLs are available from SAP as part of **R3DLLINST.zip**. 

   This ZIP file contains Microsoft run-time DLLs, which you can download from the [SAP ONE Support Launchpad website (sign-in required)](https://accounts.sap.com/saml2/idp/sso). This link has an “Attachments” option from where you can download the package. For more information, see SNOTE<sup>1</sup> 684106. This link has an “Attachments” option from where you can download the package.
```

> - If you use SAP Secure Network Communications (SNC) to connect to an SAP system, you must also have the relevant DLLs from SAP. These DLLs are different for 32-bit and 64-bit platforms and are available with SNOTE<sup>1</sup> 352295. You can download the DLLs from [here](https://go.microsoft.com/fwlink/?LinkId=104032). This link has an “Attachments” option from where you can download the package. The names of the DLLs are:

Reviewer: ```suggestion
- If you use SAP Secure Network Communications (SNC) to connect to an SAP system, you must also have the relevant DLLs from SAP, which you can find at [SAP for Me, Software Download Center (SAP sign-in required)](https://me.sap.com/softwarecenter). These DLLs are different for 32-bit and 64-bit platforms and are available with SNOTE<sup>1</sup> 352295. This link has an “Attachments” option from where you can download the package. The names of the DLLs are:
```

>   - **For 32-bit**: *gsskrb5.dll, gssntlm.dll*

Reviewer: ```suggestion
  - For 32-bit: **gsskrb5.dll**, gssntlm.dll**
```

>   - **For 64-bit x86**: *gx64krb5.dll, gx64ntlm.dll*

Reviewer: ```suggestion
  - For 64-bit x86: **gx64krb5.dll**, **gx64ntlm.dll**
```

#### 7.0 SAP client version required drivers

> - **SAP RFC SDK 7.00 UNICODE**. This is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available [here](https://go.microsoft.com/fwlink/?LinkId=94691). After you have downloaded and extracted the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders and to the relevant location mentioned preceding this table.

Reviewer: ```suggestion
- **SAP RFC SDK 7.00 UNICODE**. This is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available at the [SAP Support - Software Downloads](https://support.sap.com/en/my-support/software-downloads.html). After you download and extract the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders and to the relevant location mentioned preceding this table.
```

> - DLLs are available from SAP as part of **R3DLLINST.zip**. This contains Microsoft run-time DLLs and can be downloaded from the SAP site. See SNOTE<sup>1</sup> 684106 for more information. You can download the .zip file from [here](https://go.microsoft.com/fwlink/?LinkId=94693). This link has an “Attachments” option from where you can download the package.

Reviewer: ```suggestion
- DLLs are available from SAP as part of **R3DLLINST.zip**.

   This ZIP file contains Microsoft run-time DLLs, which you can download from [SAP Support - Software Downloads](https://support.sap.com/en/my-support/software-downloads.html). For more information, see SNOTE<sup>1</sup> 684106. This link has an “Attachments” option from where you can download the package.
```

- If you use SAP Secure Network Communications (SNC) to connect to an SAP system, you must also have the relevant DLLs from SAP. These DLLs are different for 32-bit and 64-bit platforms and are available with SNOTE<sup>1</sup> 352295. You can download the DLLs from [here](https://go.microsoft.com/fwlink/?LinkId=104032). This link has an “Attachments” option from where you can download the package. The names of the DLLs are:

>   - **For 32-bit**: *gsskrb5.dll, gssntlm.dll*

Reviewer: ```suggestion
  - For 32-bit: **gsskrb5.dll**, gssntlm.dll**
```

>   - **For 64-bit x86**: *gx64krb5.dll, gx64ntlm.dll*

Reviewer: ```suggestion
  - For 64-bit x86: **gx64krb5.dll**, **gx64ntlm.dll**
```

#### 7.1 SAP client version required drivers

> - **SAP RFC SDK 7.10 UNICODE**. This is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available [here](https://go.microsoft.com/fwlink/?LinkId=94691). After you have downloaded and extracted the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders to the relevant location mentioned preceding this table.

Reviewer: ```suggestion
- **SAP RFC SDK 7.10 UNICODE**

   This SDK is available as part of SNOTE<sup>1</sup> 27517. The instructions to download the SDK are available at the [SAP Support - Software Downloads](https://support.sap.com/en/my-support/software-downloads.html). After you download and extract the SDK, copy all the DLLs from the \rfcsdk\bin and \rfcsdk\lib folders and to the relevant location mentioned preceding this table.
```

> - DLLs are available from SAP as part of **R3DLLINST.zip**. This contains Microsoft run-time DLLs and can be downloaded from the SAP site. See SNOTE<sup>1</sup> 684106 for more information. You can download the .zip file from [here](https://go.microsoft.com/fwlink/?LinkId=94693). This link has an “Attachments” option from where you can download the package.

Reviewer: ```suggestion
- DLLs are available from SAP as part of **R3DLLINST.zip**.

   This ZIP file contains Microsoft run-time DLLs, which you can download from [SAP Support - Software Downloads](https://support.sap.com/en/my-support/software-downloads.html). For more information, see SNOTE<sup>1</sup> 684106. This link has an “Attachments” option from where you can download the package.
```

- Microsoft Visual C++ run-time DLLs required for SAP 7.1 client are available from the following links:

>   - **For 32-bit SAP 7.1 client**:  Vcredist_x86.exe [here](https://go.microsoft.com/fwlink/?LinkId=107086)

Reviewer: ```suggestion
  - For 32-bit SAP 7.1 client:  Download **Vcredist_x86.exe** from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=107086).
```

>   - **For 64-bit SAP 7.1 client**: Vcredist_x64.exe [here](https://go.microsoft.com/fwlink/?LinkId=107087)

Reviewer: ```suggestion
  - For 64-bit SAP 7.1 client: Download **Vcredist_x64.exe** from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=107087)
```

>      > [!IMPORTANT]

Reviewer: ```suggestion
       > [!IMPORTANT]
```

>      >  While installing the [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)] Evaluation version in silent mode, the default option for CEIP is true.

Reviewer: ```suggestion
       >  When you install the [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)] Evaluation version in silent mode, the default option for CEIP is **true**.
```

>      For more information about the msiexec command type `msiexec` on the command line and press `ENTER`. Or go to [here](https://go.microsoft.com/fwlink/p/?LinkId=103199).

Reviewer: The FW link doesn't go to anything:

```suggestion
     For more information about the **msiexec** command, type `msiexec` on the command line, and press `ENTER`. For more information, see [**msiexec**](/windows-server/administration/windows-commands/msiexec).
```

   msiexec /i AdaptersSetup.msi /qn /f
   ```

>     You can use various different combinations with the /f option. For more information about the msiexec command type `msiexec` on the command line, and press `ENTER`. Or go to [here](https://go.microsoft.com/fwlink/p/?LinkId=103199).

Reviewer: ```suggestion
    You can use various different combinations with the /f option. For more information about the **msiexec** command, type `msiexec` on the command line, and press `ENTER`. For more information, see [**msiexec**](/windows-server/administration/windows-commands/msiexec).
    
    
```

   > [!IMPORTANT]

>    >  While modifying the [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)] installation in the silent mode, you cannot change your preferences for opting in or out of CEIP. The preferences you chose during the installation remains, even if you explicitly set the CEIP_OPTIN to true or false.

Reviewer: ```suggestion
   > When you modify the [!INCLUDE[adapterpacknoversion](../includes/adapterpacknoversion-md.md)] installation in silent mode, you can't change your preferences for opting in or out of CEIP. The preferences you chose during the installation remains, even if you explicitly set the CEIP_OPTIN to true or false.
```

   msiexec /x AdaptersSetup.msi /qn
   ```

>     For more information about the msiexec command type `msiexec` on the command line, and press `ENTER`. Or go to [here](https://go.microsoft.com/fwlink/p/?LinkId=103199).

Reviewer: ```suggestion
   For more information about the **msiexec** command, type `msiexec` on the command line, and press `ENTER`. For more information, see [**msiexec**](/windows-server/administration/windows-commands/msiexec).
```
